sonar-project.properties

```diff
@@ -0,0 +1,10 @@
+sonar.projectKey=compound-web
+sonar.organization=vector-im
+
+
+# sonar.sources=src,res
+# sonar.tests=test,cypress
+sonar.exclusions=__test__,docs
+
+sonar.typescript.tsconfigPath=./tsconfig.json
+sonar.javascript.lcov.reportPaths=coverage/lcov.info
```

Comment:
this seems unused given the above coverage artifact is uploaded but never downloaded